### ISSUE TITLE
Support on-demand image build in Live mode.

### ISSUE THREAD

**Issue by YevheniiSemendiak** [neuro-inc/neuro-flow#814]
Desired behavior:
If one launches the job with the internal image, NF checks whether the needed image (+tag) exists.
If it does exist - proceed with the job run.
If does not exist - NF first laucnes `neuro-flow image build` for the needed image.

wdyt @mariyadavydova, @andriihomiak , @anayden , @romasku?

**Comment by anayden** [comment]
That sounds reasonable, but might be unexpected for the user. Maybe add a flag (and/or put corresponding flag to the yaml file or neuro.toml)?

**Comment by YevheniiSemendiak** [comment]
Yeah, we could add a flag to the yaml config, which by default enables build (to match the behavior of batch mode).
